app/controllers/candidate_interface/restructured_work_history/job_controller.rb

```diff
@@ -0,0 +1,81 @@
+module CandidateInterface
+  class RestructuredWorkHistory::JobController < RestructuredWorkHistory::BaseController
+    before_action :redirect_to_dashboard_if_submitted
+
+    def new
+      @job_form = RestructuredWorkHistory::JobForm.new
+    end
+
+    def create
+      @job_form = RestructuredWorkHistory::JobForm.new(job_form_params)
+
+      if @job_form.blank_form?
```

Comment:
I don't see any harm in redirecting blank forms, but I'm wondering if this is still needed as it was put in place to deal with the previous `add another` mechanism (people were clicking yes on add another job and had no way to get back to the review page)

Reply:
Ah yes - that correct. I'll remove.